ios/src/uikitMain/kotlin/main.kt

```diff
@@ -0,0 +1,59 @@
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.Surface
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.window.Application
+import com.xacalet.minesweeper.common.ui.MainScreen
+import com.xacalet.minesweeper.common.ui.theme.background
+import kotlinx.cinterop.ObjCObjectBase
+import kotlinx.cinterop.autoreleasepool
+import kotlinx.cinterop.cstr
+import kotlinx.cinterop.memScoped
+import kotlinx.cinterop.toCValues
+import platform.Foundation.NSStringFromClass
+import platform.UIKit.UIApplication
+import platform.UIKit.UIApplicationDelegateProtocol
+import platform.UIKit.UIApplicationDelegateProtocolMeta
+import platform.UIKit.UIApplicationMain
+import platform.UIKit.UIResponder
+import platform.UIKit.UIResponderMeta
+import platform.UIKit.UIScreen
+import platform.UIKit.UIWindow
+
+fun main() {
+    val args = emptyArray<String>()
+    memScoped {
+        val argc = args.size + 1
+        val argv = (arrayOf("skikoApp") + args).map { it.cstr.ptr }.toCValues()
+        autoreleasepool {
+            UIApplicationMain(argc, argv, null, NSStringFromClass(SkikoAppDelegate))
+        }
+    }
+}
+
+class SkikoAppDelegate @ObjCObjectBase.OverrideInit constructor() : UIResponder(),
+    UIApplicationDelegateProtocol {
+    companion object : UIResponderMeta(), UIApplicationDelegateProtocolMeta
+
+    private var _window: UIWindow? = null
+    override fun window() = _window
+    override fun setWindow(window: UIWindow?) {
+        _window = window
+    }
+
+    override fun application(
+        application: UIApplication,
+        didFinishLaunchingWithOptions: Map<Any?, *>?
+    ): Boolean {
+        window = UIWindow(frame = UIScreen.mainScreen.bounds)
+        window!!.rootViewController = Application("Minesweeper") {
+            Surface(
+                modifier = Modifier.fillMaxSize(),
+                color = background
+            ) {
+                MainScreen()
+            }
```

Comment:
Move Surface call to MainScreen as well as in Android application to just invoke MainScreen()